eoscostarica/guide.eoscostarica.io
# Update Boilerplates List

### Title

### What does this PR do?

- resolves https://github.com/eoscostarica/full-stack-boilerplate/issues/41

### Steps to test
1. Navigate to the tutorials section
2. See Run the boilerplate section on the sidebar

#### CheckList
